Themis/Models/Assessment.swift

```diff
@@ -5,29 +5,35 @@
 //  Created by Andreas Cselovszky on 14.11.22.
 //
 
-// swiftlint:disable line_length
-
 import Foundation
 
 enum FeedbackType {
     case inline
     case general
 }
 
-struct AssessmentResult: Encodable {
+class AssessmentResult: Encodable, ObservableObject {
```

Comment:
It got changed to a class here

Reply:
This change was made on purpose to use the UndoManager

Reply:
I think this change should be fine, but then you need to observe it like I said

Reply:
The ListView must observe it then as well!

Reply:
I presume you are referring to the FeedbackListView? I'll look into this, thanks for the help and feedback! :)